src/qibocal/protocols/state_tomography.py

```
@@ -72,10 +72,11 @@ def save(self, path):
 
     @classmethod
     def load(cls, path):
-        circuit = Circuit.from_dict(json.loads((path / CIRCUIT_PATH).read_text()))
-        data = super().load_data(path, DATAFILE)
-        params = super().load_params(path, DATAFILE)
-        return cls(data=data, circuit=circuit, targets=params["targets"])
+        if (path / CIRCUIT_PATH).exists():
```

Is it possible that it does not exist? In case, you should raise an error.

Instead, with the current arrangement you're just returning `None`, because the only `return` statement is scoped within the `if` branch.